reference/test/preconditioner/jacobi_kernels.cpp

```diff
@@ -326,6 +327,124 @@ TYPED_TEST(Jacobi, InvertsDiagonalBlocksWithAdaptivePrecision)
 }
 
 
+TYPED_TEST(Jacobi, CanTransposeDiagonalBlocks)
+{
+    using T = typename TestFixture::value_type;
+    using Bj = typename TestFixture::Bj;
+    auto tmp_bj = this->bj_factory->generate(this->mtx);
+    auto bj = gko::as<Bj>(tmp_bj->transpose());
+
+    auto scheme = bj->get_storage_scheme();
+    auto p = scheme.get_stride();
+    auto b1 = bj->get_blocks() + scheme.get_global_block_offset(0);
+    GKO_EXPECT_NEAR(b1[0 + 0 * p], T{4.0 / 14.0}, r<T>::value);
+    GKO_EXPECT_NEAR(b1[1 + 0 * p], T{2.0 / 14.0}, r<T>::value);
+    GKO_EXPECT_NEAR(b1[0 + 1 * p], T{1.0 / 14.0}, r<T>::value);
+    GKO_EXPECT_NEAR(b1[1 + 1 * p], T{4.0 / 14.0}, r<T>::value);
+
```

Comment:
this empty line should be removed since the `auto b2` below is already making it obvious that a different block is tesed.
```suggestion
```